apps/designer/app/canvas/canvas.tsx

```diff
@@ -56,6 +53,101 @@ import {
   config,
 } from "~/canvas/features/wrapper-component/text-editor";
 
+// TODO: move this somewhere else
+import { useDrag } from "~/shared/design-system/components/primitives/dnd/use-drag";
```

Comment:
lets export those from components/index so we don't have to reach into the inner folders